db/models/sector.py

```diff
@@ -9,10 +10,11 @@ class Sector(Base):
     __tablename__ = 'sector'
 
     id = Column(String, default=uuid_str, primary_key=True)
-    data = Column(LargeBinary)
+    data = Column(String)
     order = Column(Integer)
-
+    is_used = Column(Boolean, default=False)
     file_id = Column(String, ForeignKey('file.id'))
 
     def __repr__(self):
-        return f'Sector at {self.id} memory address, order is {self.order}'
+        return f'sector{self.order}'
```

Comment:
Could make it a bit more verbose:

`
f'Sector: {self.order} | {self.file.name}'
`